### DISCUSSION TITLE
Fazendo bacon com o pó da maquiagem

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#370] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/bacon-de-maquiagem/

**Comment by brunoaugusto** [comment]
Não entendi

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/videos/bacon-de-maquiagem/#comment_1087488551>):

> Não entendi

Foi um erro de digitação. Em vez de escrever \*batom\*, ela escreveu \*bacon\*

**Comment by one-eyed-willy** [comment]
![null](<https://i.postimg.cc/jjx5c4Dg/homer-drolling-bacon.png>)

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/videos/bacon-de-maquiagem/#comment_1087489714>):

> Foi um erro de digitação. Em vez de escrever \*batom\*, ela escreveu \*bacon\*

![null](<https://i.postimg.cc/fym037qw/monica-ata.png>)
